db/data.py

```diff
@@ -460,3 +461,32 @@ def get_summary_data(mbid, offset=0):
         pass
 
     return summary
+
+def check_low_level_duplicates(data):
+    """
+        Check if data comes from an audio file already analysed
+        Uses the md5_encoded field
+    Args: data
+        A dictionary containing the low-level data extracted using essentia
+    """
+    if 'md5_encoded' in data['metadata']['audio_properties']:
+        input_md5 = data['metadata']['audio_properties']['md5_encoded']
+        query = text("select data->'metadata'->'tags'->'musicbrainz_recordingid' from lowlevel_json where data->'metadata'->'audio_properties'->'md5_encoded' ? :md5 ")
```

Comment:
You can get the recording ID from the lowlevel table here. You should join it on the id field of `lowlevel` and `lowlevel_json`